Dockerfile

```diff
@@ -1,23 +1,32 @@
 # Docker Image for BuildKite CI
 # -----------------------------
```

Comment:
Should we change the comment to mention `tile-converter`. Can we add more text here describing what this is and how it is used? Most JS maintainers do not know about docker files.

Reply:
Added description. How to build image and how to use it.